src/components/metadata.svelte

```diff
@@ -1,20 +1,33 @@
 <script lang="ts">
-  import { MetaTags, type MetaTagsProps } from 'svelte-meta-tags';
-  import type { Seo } from '../data/types';
+  import { MetaTags } from 'svelte-meta-tags';
   import dedent from 'dedent';
   import { urlFor } from '../data/sanity-client-browser';
   import type { Picture } from 'vite-imagetools';
+  import type { Metadata } from '../data/types';
 
```

Comment:
```suggestion
   export let title = string | undefined = undefined;
   export let searchEngine = SearchEngineMetadata | undefined = undefined;   
```

Can we do this instead?